src/index.js

```diff
@@ -44,18 +64,37 @@ export default function (...args) {
   /**
    * @param {{id: string}[]} existing
    * @param {{id: string}[]} descriptors
+   * @param {string} filename
+   * @param {boolean} cleanUpUnusedMessages
    *
    * @returns {{id: string}[]}
    */
-  function mergeDescriptors (existing, descriptors) {
+  function mergeDescriptors(cleanUpUnusedMessages, existing, descriptors, filename) {
+    if (!cleanUpUnusedMessages) {
```

Comment:
It's a bad practice to use flags like this. You can make your own method, and decide which of methods should be called in src/index.js:135

Reply:
I personally don't like it too. I just followed existing patterns in the code, replicating the behavior in `mergeMessages` method.

When refactoring the method, should I update `mergeMessages` method too?

Reply:
Done.